app/client/components/identity/EmailAndMarketing/OptOutSection.tsx

```diff
@@ -12,30 +12,49 @@ interface OptOutSectionProps {
   clickHandler: ClickHandler;
 }
 
-const optOutFinder = (
-  consents: ConsentOption[],
-  clickHandler: ClickHandler
-) => (id: string) => {
-  const consent = consents.find(c => c.id === id);
-  return (
-    consent && (
-      <MarketingPreference
-        id={consent.id}
-        description={consent.description}
-        selected={consent.subscribed}
-        onClick={clickHandler}
-      />
-    )
-  );
-};
+/**
+ * NOTE:
+ * Only use this method for an OPT OUT consent, eg. "post_optout"
+ * The description of Opt On consents have changed so for UX/UI purposes they are now opt INs
+ * The backend model remains an opt OUT, so we invert the consented/subscribed value here.
+ */
+const optOutFinderAndInverter =
+  (consents: ConsentOption[], clickHandler: ClickHandler) =>
+  (id: string, altDescription?: string) => {
+    const consent = consents.find((c) => c.id === id);
+    return (
+      consent && (
+        <MarketingToggle
+          id={consent.id}
+          title={consent.name}
+          description={consent.description || altDescription} // Not all consents from IDAPI have a description
+          selected={!consent.subscribed} // Opt Out consent value is inverted
+          onClick={clickHandler}
+        />
+      )
+    );
+  };
 
-const standardTextSize = {
-  fontSize: "17px"
-};
+const YourDataDescription: FC = () => (
```

Comment:
I'm getting a React warning in the console here: ` validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.` Any recommendations?

Reply:
I assume that where this component is being used is under a `<p>` tag, in turn causing the `<ul>` to appear as a child of `<p>`.

Ideally you'd want to change where this component is inserted to be a `<div>`, or if that's not possible then insert this component in a different way.